src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -68,7 +68,7 @@ const MilitaryInformationContent = ({ militaryInformation }) => {
           }
         />
       );
-    } else if (some(error.errors, e => ['500', '503'].includes(e.code))) {
```

Comment:
It doesn't make sense to only show this fallback error if the code is 500 or 503. If there is _any_ response error that we don't have special handling for, show this fallback `LoadFail` component.